slick-testkit/src/codegen/scala/slick/test/codegen/GenerateMainSources.scala

```diff
@@ -143,13 +143,16 @@ val  SimpleA = CustomTyping.SimpleA
           |  import java.util.UUID
           |  val u1 = UUID.randomUUID()
           |  val u2 = UUID.randomUUID()
-          |  val p1 = PersonRow(1, u1)
-          |  val p2 = PersonRow(2, u2)
+          |  val u3 = UUID.randomUUID()
+          |  val u4 = UUID.randomUUID()
+          |  val p1 = PersonRow(1, u1, uuidFunc = Some(u3))
+          |  val p2 = PersonRow(2, u2, uuidFunc = Some(u4))
```

Comment:
There should now be both uuidFunc and uuidDef in the generated code. I wrote the test as failing and then fixed the code, and it did initially fail when I ran testkit/test.

Note that it seems to sometimes be necessary to run testkit/clean before testkit/test - I don't think all the dependencies used by the code generator are explicitly specified - so you may be looking at an old version of the generated code?